apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -12,82 +13,70 @@ const InvertedWrapper: React.FC = ({ children }) => {
 };
 
 storiesOf('Spinner converged', module)
-  .addDecorator(TestWrapperDecoratorNoAnimation)
-  .addStory('Primary', () => <Spinner className="test-class" />, {
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(PauseAnimationDecorator)
+  .addDecorator(story => (
+    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
+  ))
+  .addStory('Primary', () => <Spinner />, {
     includeHighContrast: true,
     includeDarkMode: true,
   })
-  .addStory('Primary with Label', () => <Spinner className="test-class" label="Loading" />, {
+  .addStory('Primary with Label', () => <Spinner label="Loading" />, {
     includeHighContrast: true,
     includeDarkMode: true,
   })
-  .addStory(
-    'Primary with Label Before',
-    () => <Spinner className="test-class" labelPosition="before" label="Loading" />,
-    {
-      includeHighContrast: true,
-      includeDarkMode: true,
-      includeRtl: true,
-    },
-  )
-  .addStory(
-    'Primary with Label After',
-    () => <Spinner className="test-class" labelPosition="after" label="Loading" />,
-    {
-      includeHighContrast: true,
-      includeDarkMode: true,
-      includeRtl: true,
-    },
-  )
-  .addStory(
-    'Primary with Label Above',
-    () => <Spinner className="test-class" labelPosition="above" label="Loading" />,
-    {
-      includeHighContrast: true,
-      includeDarkMode: true,
-    },
-  )
-  .addStory(
-    'Primary with Label Below',
-    () => <Spinner className="test-class" labelPosition="below" label="Loading" />,
-    {
-      includeHighContrast: true,
-      includeDarkMode: true,
-    },
-  )
-  .addStory('Primary with Size Tiny', () => <Spinner className="test-class" size="tiny" />, {
+  .addStory('Primary with Label Before', () => <Spinner labelPosition="before" label="Loading" />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+    includeRtl: true,
+  })
+  .addStory('Primary with Label After', () => <Spinner labelPosition="after" label="Loading" />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+    includeRtl: true,
+  })
+  .addStory('Primary with Label Above', () => <Spinner labelPosition="above" label="Loading" />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Primary with Label Below', () => <Spinner labelPosition="below" label="Loading" />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
```

Comment:
Could we add all of these into one story?
```suggestion
  .addStory('Primary', () => {
    <div style={{ display: 'flex', flexDirection: 'column', rowGap: '20px'}}>
      <Spinner labelPosition="before" label="Loading" />
      <Spinner labelPosition="after" label="Loading" />
      <Spinner labelPosition="above" label="Loading" />
    </div>
    }, {
    includeHighContrast: true,
    includeDarkMode: true,
    includeRtl: true,
  })
  .addStory('Primary + size', () => {
    <div style={{ display: 'flex', flexDirection: 'column', rowGap: '20px'}}>
      {/* All the Spinners with unique size go here */}
    </div>
    }, {
    includeHighContrast: true,
    includeDarkMode: true,
    includeRtl: true,
  })
```